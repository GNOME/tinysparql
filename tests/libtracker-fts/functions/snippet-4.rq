SELECT ?u fts:snippet(?u, '>>>', '<<<', '...', 0) { ?u fts:match 'bananas' } order by desc fts:rank(?u)
