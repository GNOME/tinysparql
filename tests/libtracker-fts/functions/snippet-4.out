"http://www.example.org/test#1"	
"http://www.example.org/test#2"	">>>bananas<<< >>>bananas<<<..."
"http://www.example.org/test#4"	">>>bananas<<<..."
"http://www.example.org/test#3"	">>>bananas<<<..."
